lib/appsignal/config.ex

```diff
@@ -185,6 +187,7 @@ defmodule Appsignal.Config do
       System.put_env("_APPSIGNAL_HTTP_PROXY", config[:http_proxy])
     end
     System.put_env("_APPSIGNAL_IGNORE_ACTIONS", config[:ignore_actions] |> Enum.join(","))
+    System.put_env("_APPSIGNAL_IGNORE_COOKIES", config[:ignore_cookies] |> Enum.join(","))
```

Comment:
makes sense, if the agent doesn't log which config vars are used for debugging purposes. r
removed it.